src/modules/universe/RadixUniverse.ts

```diff
@@ -77,7 +77,9 @@ export default class RadixUniverse {
 
         // Find native token
         for (const atom of this.universeConfig.genesis) {
-            const tokenClasses = atom.getParticlesOfType(RadixTokenDefinitionParticle)
+            const tokenClasses = []
+                .concat(atom.getParticlesOfType(RadixFixedSupplyTokenDefinitionParticle))
+                .concat(atom.getParticlesOfType(RadixMutableSupplyTokenDefinitionParticle))
```

Comment:
Hmm... Java lib does actually not look for `MutableSupplyTokens` amongst the genesis atoms, so I too just looked for `FixedSupplyTokens`, we need to consolidate this :D